src/data/logoIcons.json

```diff
@@ -302,7 +302,7 @@
     {
       "name": "tail-9X",
       "title": "Airplane tail image for Mokulele Airlines",
-      "viewBox": "0 0 856 830"
+      "viewBox": "0 0 736 707"
```

Comment:
All other tail logos have the `0 0 856 830` value. Why is this the only tail that is different?

Reply:
That was my bad. I fixed the SVG data but didn't push the branch. Oops.